newIDE/app/src/EventsSheet/ParameterFields/LayerField.js

```diff
@@ -1,42 +1,135 @@
 // @flow
-import React, { Component } from 'react';
+import React from 'react';
+import { t } from '@lingui/macro';
+import { I18n } from '@lingui/react';
 import { mapFor } from '../../Utils/MapFor';
-import { type ParameterFieldProps } from './ParameterFieldCommons';
+import {
+  type ParameterFieldProps,
+  type ParameterFieldInterface,
+  type FieldFocusFunction,
+} from './ParameterFieldCommons';
+import SelectField, { type SelectFieldInterface } from '../../UI/SelectField';
+
 import GenericExpressionField from './GenericExpressionField';
-import { type ExpressionAutocompletion } from '../../ExpressionAutocompletion';
+import SelectOption from '../../UI/SelectOption';
+import { TextFieldWithButtonLayout } from '../../UI/Layout';
+import OpenInNew from '@material-ui/icons/OpenInNew';
```

Comment:
I'm not sure it's the best icon here. It's used for the leaderboard admin and it makes sense since it opens the admin, but here it doesn't make great sense to me

Reply:
yeah agree, I'll see if we can find something better